vendor/src/golang.org/x/crypto/ssh/agent/example_test.go

```diff
@@ -6,20 +6,20 @@ package agent_test
 
 import (
 	"log"
-	"os"
 	"net"
+	"os"
 
-        "golang.org/x/crypto/ssh"
```

Comment:
This does not belong into the PR, did you accidentally run `gofmt` on the `vendor` directory?